gpt_index/indices/query/vector_store/base.py

```diff
@@ -18,8 +18,10 @@ def __init__(
         index_struct: BID,
         embed_model: Optional[BaseEmbedding] = None,
         similarity_top_k: Optional[int] = 1,
+        similarity_cutoff: Optional[float] = None,
```

Comment:
I'm probably ok setting this on the base class actually (and raise an error in _should_use_node if similarity is None and similarity_cutoff exists)

Reply:
when you say base class, you mean `BaseGPTIndex` right?

Reply:
BaseGPTIndexQuery